.github/workflows/main.yml

```diff
@@ -37,8 +37,21 @@ jobs:
 
       - name: Run performance tests
         run: cd examples/native && ./reassure-tests.sh
-
-      - name: Run Danger.js
-        run: yarn danger ci
+          
+  build-validate-pref-tests-with-custom-action:
+    runs-on: ubuntu-latest
+    name: A job to test the new custom GH action
+    steps:
+      - uses: actions/checkout@v3
+        with:
```

Comment:
@ShaswatPrabhat Seems like using your ref is causing the related step to fail could you omit the `ref` options?

Reply:
I had tried earlier with the same, let me try again @mdjastrzebski

Reply:
Got [this](https://github.com/callstack/reassure/actions/runs/3360152110/jobs/5568998534) now

Reply:
@mdjastrzebski might need some help here, whenever you have some time. Thanks!